releases/release-5.0.0.md

```diff
@@ -0,0 +1,476 @@
+- Support the case-insensitive comparison sort for collations
+
+### Security
+
+[User document](/log-redaction.md), [#18566](https://github.com/pingcap/tidb/issues/18566)
+
+To meet security compliance requirements (such as *General Data Protection Regulation*, or GDPR), the system supports desensitizing information (such as ID and credit card number) in the output error messages and logs, which can avoid leaking sensitive information.
+
+TiDB supports desensitizing the output log information. To enable this feature, use the following switches:
+
++ The global variable [`tidb_redact_log`](/system-variables.md#tidb_redact_log). Its default value is `0`, which means that desensitization is disabled. To enable desensitization for tidb-server logs, set the variable value to `1`.
++ The configuration item `security.redact-info-log`. Its default value is `false`, which means that desensitization is disabled. To enable desensitization for tikv-server logs, set the variable value to `true`.
++ The configuration item `security.redact-info-log`. Its default value is `false`, which means that desensitization is disabled. To enable desensitization for pd-server logs, set the variable value to `true`.
++ The configuration item `security.redact_info_log` for tiflash-server and `security.redact-info-log` for tiflash-learner. Their default values are both `false`, which means that desensitization is disabled. To enable desensitization for tiflash-server and tiflash-learner logs, set the values of both variables to `true`.
+
+
+This feature is introduced in v5.0. To use the feature, enable the system variable and all configuration items above.
+
+## Performance optimization
+
+### MPP architecture
+
+[User document](/tiflash/use-tiflash.md)
+
+TiDB introduces the MPP architecture through TiFlash nodes. This architecture allows multiple TiFlash nodes to share the execution workload of large join queries.
+
+When the MPP mode is on, TiDB determines whether to send a query to the MPP engine for computation based on the calculation cost. In the MPP mode, TiDB distributes the computation of table joins to each running TiFlash node by redistributing the join key during data calculation (`Exchange` operation), and thus accelerates the calculation. Furthermore, with the aggregation computing feature that TiFlash has already supported, TiDB can pushdown the computation of a query to the TiFlash MPP cluster. Then the distributed environment can help accelerate the entire execution process and dramatically increase the speed of analytic queries.
+
+In the benchmark test on three 40C 189G physical machines at the scale of TPC-H 100, TiFlash MPP delivers significant processing speed over analytic engines of traditional analytic databases and data lakes. With this architecture, you can perform large-scale analytic queries directly on the latest transaction data, with a higher performance than traditional offline analytic solutions.

```

Comment:
```suggestion
In the TPC-H 100 benchmark test, TiFlash MPP delivers significant processing speed over analytic engines of traditional analytic databases and SQL on Hadoop. With this architecture, you can perform large-scale analytic queries directly on the latest transaction data, with a higher performance than traditional offline analytic solutions.

```

Reply:
Here we don't need to expose machine details since all other important details are not listed in this brief.

Reply:
I've also PRed directly in Chinese docs.